Mathlib/Mathport/Notation.lean

```diff
@@ -587,7 +587,8 @@ elab (name := notation3) doc:(docComment)? attrs?:(Parser.Term.attributes)? attr
       let delabKeys := ms.foldr (·.1 ++ ·) []
       trace[notation3] "Adding `delab` attribute for keys {delabKeys}"
       for key in delabKeys do
-        elabCommand <| ← `(command| attribute [delab $(mkIdent key)] $(Lean.mkIdent delabName))
+        elabCommand <|
+          ← `(command| attribute [$attrKind delab $(mkIdent key)] $(Lean.mkIdent delabName))
```

Comment:
@kmill, can you check this looks sensible?

Reply:
Yes, I think so, and I think this must have been an oversight on my part since I was under the impression that delaborators were scoped if the notation was.

Reply:
Split to #8934, since I think this fix is more pressing than the notation which might go through more bikeshedding.